central/node/datastore/store/common/v2/parts_test.go

```diff
@@ -200,7 +200,21 @@ func TestSplitAndMergeNode(t *testing.T) {
 	}
 
 	splitActual := Split(node, true)
-	assert.Equal(t, splitExpected, splitActual)
+	protoassert.Equal(t, splitExpected.Node, splitActual.Node)
+
+	assert.Len(t, splitActual.Children, len(splitActual.Children))
+	for i, expected := range splitExpected.Children {
+		actual := splitActual.Children[i]
+		protoassert.Equal(t, expected.Component, actual.Component)
+		protoassert.Equal(t, expected.Edge, actual.Edge)
+
+		assert.Len(t, actual.Children, len(expected.Children))
+		for i, expected := range expected.Children {
```

Comment:
nitpick: It would be good if we could avoid shadowing outer declarations. Maybe we can prefix variable names with `inner*`, `sub*` or something like that.

Same comment for `central/image/datastore/store/common/v2/parts_test.go`